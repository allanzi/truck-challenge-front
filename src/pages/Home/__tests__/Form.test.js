import React from 'react';

import FormPage from '../Form';

it('should render correct', () => {
  const wrapper = shallow(<FormPage />);

  expect(wrapper).toMatchSnapshot();
});
