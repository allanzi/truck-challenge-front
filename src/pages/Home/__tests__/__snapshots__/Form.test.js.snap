// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correct 1`] = `
<Modal
  cancelText="Cancelar"
  confirmLoading={false}
  maskTransitionName="fade"
  okText="Cadastrar"
  okType="primary"
  onCancel={[Function]}
  onOk={[Function]}
  title="Cadastrar Terminal"
  transitionName="zoom"
  visible={true}
  width={520}
>
  <ForwardRef(InternalForm)
    form={
      Object {
        "__INTERNAL__": Object {},
        "getFieldError": [Function],
        "getFieldValue": [Function],
        "getFieldsError": [Function],
        "getFieldsValue": [Function],
        "getInternalHooks": [Function],
        "isFieldTouched": [Function],
        "isFieldValidating": [Function],
        "isFieldsTouched": [Function],
        "isFieldsValidating": [Function],
        "resetFields": [Function],
        "scrollToField": [Function],
        "setFields": [Function],
        "setFieldsValue": [Function],
        "submit": [Function],
        "validateFields": [Function],
      }
    }
    layout="vertical"
    name="create_terminal"
  >
    <FormItem
      label="Caminhoneiro ID"
      name="user_id"
      rules={
        Array [
          Object {
            "message": "Este campo é obrigatório!",
            "required": true,
          },
        ]
      }
    >
      <Input
        type="text"
      />
    </FormItem>
    <FormItem
      label="Está carregado?"
      name="is_busy"
    >
      <Switch />
    </FormItem>
  </ForwardRef(InternalForm)>
</Modal>
`;
